managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedgerFactoryConfig.java

```diff
@@ -75,4 +75,9 @@
      * cluster name for prometheus stats
      */
     private String clusterName;
+
+    /**
+     * ManagedLedgerInfo compression type. If the compression type is null or invalid, don't compress data.
+     */
+    private String managedLedgerInfoCompressionType = null;
```

Comment:
We'd better use enum type to strict compression type info specific type instead of string. I'm not sure whether it's easy to implement.

Reply:
Using the configuration `managedLedgerInfoCompressionType` of the `ServiceConfiguration` to set this field, maybe we could convert `String` to `ComprssionType` in `MetaStoreImpl`.